addons/popochiu/editor/config/config.gd

```diff
@@ -1,127 +1,210 @@
+	_set_project_setting(INVENTORY_ITEMS_ON_START, items)
 
-func is_default_wipe_old_anims_enabled() -> bool:
-	return _get_project_setting(_DEFAULT_WIPE_OLD_ANIMS_ENABLED, true)
 
+static func get_inventory_items_on_start() -> Array:
+	return _get_project_setting(INVENTORY_ITEMS_ON_START, [])
 
 
-#endregion
+# ---- Aseprite Importing --------------------------------------------------------------------------
+static func is_default_animation_import_enabled() -> bool:
+	return _get_project_setting(ASEPRITE_IMPORT_ANIMATION, true)
+
+
+static func is_default_animation_loop_enabled() -> bool:
+	return _get_project_setting(ASEPRITE_LOOP_ANIMATION, true)
 
-#region Private ####################################################################################
-func _default_command() -> String:
-	return 'aseprite'
 
+static func is_default_animation_prop_visible() -> bool:
+	return _get_project_setting(ASEPRITE_PROPS_VISIBLE, true)
 
-func _set_icons() -> void:
-	_plugin_icons = {
-		"collapsed": ei.get_base_control().get_theme_icon("GuiTreeArrowRight", "EditorIcons"),
-		"expanded": ei.get_base_control().get_theme_icon("GuiTreeArrowDown", "EditorIcons"),
-	}
 
+static func is_default_animation_prop_clickable() -> bool:
+	return _get_project_setting(ASEPRITE_PROPS_CLICKABLE, true)
 
-func _initialize_editor_cfg(key: String, default_value, type: int, hint: int = PROPERTY_HINT_NONE):
-	if not editor_settings.has_setting(key):
-		editor_settings.set_setting(key, default_value)
-		editor_settings.set_initial_value(key, default_value, false)
-		editor_settings.add_property_info({
-			"name": key,
-			"type": type,
-			"hint": hint,
-		})
 
+static func is_default_wipe_old_anims_enabled() -> bool:
+	return _get_project_setting(ASEPRITE_WIPE_OLD_ANIMATIONS, true)
 
-func _initialize_project_cfg(key: String, default_value, type: int, hint: int = PROPERTY_HINT_NONE):
-	if not ProjectSettings.has_setting(key):
-		ProjectSettings.set_setting(key, default_value)
-		ProjectSettings.set_initial_value(key, default_value)
-		ProjectSettings.add_property_info({
-			"name": key,
-			"type": type,
-			"hint": hint,
-		})
 
+# ---- Pixel game ----------------------------------------------------------------------------------
+static func set_pixel_art_textures(use_pixel_art_textures: bool) -> void:
+	_set_project_setting(PIXEL_ART_TEXTURES, use_pixel_art_textures)
 
-func _get_editor_setting(key: String, default_value):
-	var e = editor_settings.get_setting(key)
-	return e if e != null else default_value
 
+static func is_pixel_art_textures() -> bool:
+	return _get_project_setting(PIXEL_ART_TEXTURES, false)
 
-func _get_project_setting(key: String, default_value):
+
+static func is_pixel_perfect() -> bool:
+	return _get_project_setting(PIXEL_PERFECT, false)
+
+
+#endregion
+
+#region Private ####################################################################################
+static func _initialize_project_cfg(
```

Comment:
It's just nitpicking, but what if this is called `_initialize_project_setting()` instead? I know it come in the `config.gd` file so maybe it doesn't make sense, but it would be more readable about what it does in the new context.

Reply:
I agree. I'll do the change.